Vagrantfile

```diff
@@ -77,16 +79,16 @@ Vagrant.configure("2") do |config|
       box.vm.synced_folder '.', '/vagrant', disabled: !properties[:share_cwd]
 
       # Configure winrm if booting windows machine
-      if "#{hostname}".start_with?("win")
-        # box.vm.communicator = "winrm"
-        box.winrm.username = "vagrant"
-        box.winrm.password = "vagrant"
-        # Allow to use basic auth for login
-        box.vm.provision "shell", inline: "Set-Item -Path WSMan:\\localhost\\Service\\Auth\\Basic -Value $true"
-        box.vm.provision "shell", inline: "choco feature enable -n=allowGlobalConfirmation"
-        # Install powershell 6 with linux remoting
-        box.vm.provision "shell", inline: "iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/softasap/sa-win/master/GetPowershell6LinuxRemoting.ps1'))"
-      end
+      # if "#{hostname}".start_with?("win")
```

Comment:
Are you sure one needs to be commented?

Reply:
if need this only to make ansible provisioning work, if you just want to test/install the agent is not needed, that is why I commented it out